portal-web/test/functional/com/liferay/portalweb/paths/portlet/ckeditorsample/CKEditorBalloonEditorSample.path

```diff
@@ -0,0 +1,63 @@
+<html>
+<head>
+<title>CKEditorBalloonEditorSample</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+	<tr><td rowspan="1" colspan="3">CKEditorBalloonEditorSample</td></tr>
+</thead>
+
+<tbody>
+
+<!--PARAGRAPH-->
+<tr>
+	<td>BODY_TEXT</td>
+	<td>//div[@class='portlet-body']/div/div/p[${paragraph_number}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BODY_TEXT_HYPERLINK</td>
+	<td>//div[@class='portlet-body']/div/div/p[2]/a | //div[@class='portlet-body']/div/div/p/a</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_ASSERT_BODY_TEXT_CONTENT_BOLD</td>
```

Comment:
I'm nit picking but I'd change the name to BODY_TEXT_BOLD to keep it simple name for path. The "assert paragraph" description will be better described in the macro as the function name asserting the path.

I'd also like to move this to CKEditor.path. Same for the other paths.

My vision is to make CKEditor.path locators work for the default CKEditor and available for other product teams to utilize.